packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -25,9 +25,7 @@ onMount(async () => {
 
 async function updateProxySettings() {
   await window.setProxyState(proxyState);
-  if (proxyState !== ProxyState.PROXY_SYSTEM) {
```

Comment:
Note that this condition was added by @jeffmaury a while ago to fix #9379.

This condition causes the proxy configuration not to be updated when the proxy configuration is switched to System. When I deleted this condition, the test did not find any errors.

I am not sure why this condition was added, maybe further discussion is needed.